src/main/java/Duke.java

```diff
@@ -1,10 +1,167 @@
+import java.util.ArrayList;
+import java.util.Scanner;
+
 public class Duke {
+    private static final String GREETING = "Hello! I'm Duke\nWhat can I do for you?";
+    private static final String GOODBYE = "Bye. Hope to see you again soon!";
+    private static final ArrayList<Task> TASKS = new ArrayList<>();
+
+    /**
+     * @param txt text to indent.
+     * @return indented string.
+     */
+    private static String autoIndent(String txt) {
+        return "    " + txt.replace("\n", "\n    ");
+    }
+
+    /**
+     * prints border line.
+     */
+    private static void borderLine() {
+        System.out.print("    ____________________________________________________________\n");
+    }
+
+    /**
+     * Prints output in a nice format. (adds borders and indentation).
+     *
+     * @param inp input string.
+     */
+    private static void prettifyOut(String inp) {
```

Comment:
Wouldn't naming the method as a verb such as "prettifyOutput" be more appropriate, personally it took me a while to understand what Out means